src/views/FindPurchaseOrder.vue

```diff
@@ -123,16 +123,15 @@
 
                   <div>
                     <ion-chip outline>
-                      <!-- TODO -->
+                      <!-- TODO, Update icon -->
                       <ion-icon :icon="pricetag" />
                       <ion-label>Shopify ID</ion-label>
                     </ion-chip>
                   </div>
 
-                  <div>
-                    <ion-item lines="none">
-                      <ion-note slot="end">ATP Ordered</ion-note>
-                    </ion-item>
+                  <div class="items-metadata">
+                    <ion-note slot="end">ATP</ion-note>
```

Comment:
if these ion-notes are in a div, slot end isn't doing anything right?

Reply:
Yes sir